gc/verbose/handler_standard/VerboseHandlerOutputStandard.cpp

```diff
@@ -244,6 +244,28 @@ MM_VerboseHandlerOutputStandard::getCycleType(uintptr_t type)
 	return cycleType;
 }
 
+const char *
+MM_VerboseHandlerOutputStandard::getConcurrentTypeString(uintptr_t type)
+{
+	const char* cycleType = NULL;
+	switch (type) {
+	case OMR_GC_CYCLE_TYPE_DEFAULT:
```

Comment:
We should never have default Concurrent Cycle, let's remove this. Anything outside of CS and CM should be unknown